totalRP3/Core/Enums.lua

```diff
@@ -44,3 +44,13 @@ AddOn_TotalRP3.Enums.UNIT_TYPE = {
 	MOUNT = "MOUNT", -- Not a real unit, but used for companion profiles.
 	NPC = "NPC",
 };
+
+-- Configuration --
+AddOn_TotalRP3.Enums.GENERAL_SETTINGS_CONFIG_KEYS = {
+	DATE_FORMAT = "date_format"
```

Comment:
For now I'm tempted to suggest just hardcoding the "date_format" config key in the areas it's being used; longer-term I want to rework the config key nonsense and would rather not have to _also_ factor in this enum,

Reply:
Oof, yeah it hurts to do that but I understand that my changes would add more areas to look over for your refacto !